front/src/modules/study/components/AddOrEditStudyModal.tsx

```diff
@@ -42,11 +44,29 @@ type StudyParams = {
   projectId: string;
 };
 
-const emptyStudy: StudyForm = { name: '', state: STUDY_STATES.started, tags: [] };
+const emptyStudy: StudyForm = {
+  actual_end_date: null,
+  budget: undefined,
+  business_code: '',
+  description: '',
+  expected_end_date: null,
+  name: '',
+  service_code: '',
+  start_date: null,
+  state: STUDY_STATES.started,
+  study_type: 'nothingSelected',
+  tags: [],
+};
 
 export default function AddOrEditStudyModal({ editionMode, study }: Props) {
-  const { t } = useTranslation('operationalStudies/study');
-  const { closeModal } = useContext(ModalContext);
+  const { t } = useTranslation(['operationalStudies/study', 'translation']);
```

Comment:
For the `translation` trad, as you use the whole path of the key, it seems you don't need to add it here
```suggestion
  const { t } = useTranslation('operationalStudies/study');
```

Reply:
I think it's better to keep it and remove this key from `t()`, this way the app will skip searching in every locale folders and always point to the good ones, maybe I m wrong

Reply:
for thoses 2 cases, we should use the exact same implementation for every translation, not 2 different kinds

Reply:
Yes you're right, totally agreed

Reply:
So for this comment and the others, you can keep `translation` in `useTranslation` and remove the `translation:` in the `t()` call

Reply:
fix

Reply:
With this solution :
`const { t } = useTranslation(['operationalStudies/scenario', 'translation']);`
`title={t('common.leaveEditionMode')}`
it doesn't work. 

The only ways to have the translation are : 
`const { t } = useTranslation(['operationalStudies/scenario', 'translation']);`
`title={t('translation:common.leaveEditionMode')}`

and 

`const { t } = useTranslation('operationalStudies/scenario');`
`title={t('translation:common.leaveEditionMode')}`

So what do you prefer ? Keep 'translation' in both useTranslation and call, or just in the call ?

Reply:
sorry my bad, I forgot to tell you, when you point to an array in useTranslation, you have to select the namespace if it's not at index 0 with this syntax: `title={t('common.leaveEditionMode', { ns: 'translation' })}`
[check this doc for more](https://react.i18next.com/latest/usetranslation-hook#usetranslation-params)